packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -168,6 +168,22 @@ LinkButton = () => (
 );
 ```
 
+## Inline link as span
+```js
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+
+InlineLinkAsSpan = () => (
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    <Button variant="link" isInline component="span">
+      This is long button text that needs to be a span so that it will wrap inline with the text around it.
+    </Button>
+    Sed hendrerit nisi in cursus maximus. Ut malesuada nisi turpis, in condimentum velit elementum non.
```

Comment:
Change to `{' '}Sed ...` to preserve whitespace.

Reply:
Updated